Framework/Algorithms/src/MaxEnt.cpp

```diff
@@ -20,6 +23,20 @@ using namespace Kernel;
 // Register the algorithm into the AlgorithmFactory
 DECLARE_ALGORITHM(MaxEnt)
 
+namespace {
+
+std::map<std::string, std::string> inverseCaption = {{"Time", "Frequency"},
```

Comment:
Thanks Anders. I'll add a code comment and a note to the documentation (I did add a comment to the release notes). And yes, my idea was that in the future we can add new units and labels if scientists request it. I'll also add a unit test for the label conversion.